<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TabDb_RemoveDuplicateScores</class>
 <widget class="QWidget" name="TabDb_RemoveDuplicateScores">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">TabDb_RemoveDuplicateScores</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>scan.title</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QCheckBox" name="scan_option_scoreCheckBox">
          <property name="text">
           <string>scan.option.score</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QCheckBox" name="scan_option_pureCheckBox">
            <property name="text">
             <string notr="true">PURE</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="scan_option_farCheckBox">
            <property name="text">
             <string notr="true">FAR</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="scan_option_lostCheckBox">
            <property name="text">
             <string notr="true">LOST</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QCheckBox" name="scan_option_maxRecallCheckBox">
          <property name="text">
           <string notr="true">MAX RECALL</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QCheckBox" name="scan_option_clearTypeCheckBox">
            <property name="text">
             <string>scan.option.clearType</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="scan_option_modifierCheckBox">
            <property name="text">
             <string>scan.option.modifier</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="scan_scanButton">
          <property name="text">
           <string>scan.scanButton</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>quickSelect.title</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>quickSelect.keepSingleLabel</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="quickSelect_timeComboBox"/>
        </item>
        <item>
         <widget class="QComboBox" name="quickSelect_ColumnComboBox"/>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>91</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="quickSelect_selectButton">
          <property name="text">
           <string>quickSelect.selectButton</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="collapseAllButton">
       <property name="text">
        <string>collapseAllButton</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="expandAllButton">
       <property name="text">
        <string>expandAllButton</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="resetModelButton">
       <property name="text">
        <string>resetModelButton</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <widget class="QTreeView" name="treeView">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::NoSelection</enum>
     </property>
     <property name="verticalScrollMode">
      <enum>QAbstractItemView::ScrollPerPixel</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="deleteSelectedButton">
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton { color: red };</string>
     </property>
     <property name="text">
      <string>deleteSelectedButton</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
